src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -172,62 +172,62 @@ private String getDropdownMenuLabel(FeedbackQuestionAttributes question) {
     }
 
     private boolean isVisibilitySetToAnonymousToRecipientAndInstructors(FeedbackQuestionAttributes question) {
-        boolean responsesVisibleOnlyToRecipientAndInstructors = question.showResponsesTo.size() == 2
+        boolean isResponsesVisibleOnlyToRecipientAndInstructors = question.showResponsesTo.size() == 2
```

Comment:
I think it should be read as this `is` `ResponsesVisibleOnlyToRecipientAndInstructors` (`true` or `false`). Use `is` for consistency. So next time when we see `is`, we would know it is a boolean variable.